index.js

```diff
@@ -1,18 +1,50 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+let header = document.querySelector("h1#header")
+console.log("Here's your header:", header.textContent)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+let travelerSpecs = document.querySelector("#profile")
+h2ToUpdate = travelerSpecs.querySelector("h2")
+h2ToUpdate.textContent = traveler.name
 
-/***** Deliverable 4 *****/
+emToUpdate = travelerSpecs.querySelector("em")
+emToUpdate.textContent = traveler.nickname
+
+imgToUpdate = travelerSpecs.querySelector("img")
+imgToUpdate.src = traveler.photo
+imgToUpdate.alt = traveler.name
```

Comment:
Make sure to always use `let` or `const` when declaring variables -- you did it correctly for `travelerSpecs`, but your variables for `h2ToUpdate`, `emToUpdate`, and `imgToUpdate` also need `let` when you declare the variable.